client/src/views/Home.js

```diff
@@ -0,0 +1,47 @@
+import React, {Component} from 'react';
+import {View, Text, Button, StyleSheet} from 'react-native';
+import {navigateTo} from '../helpers/NavigateHelper';
+
+class Home extends Component {
+  render() {
+    const {navigation} = this.props;
+    return (
+      <View style={styles.container}>
+        <Text style={styles.text}>We are at home screen.</Text>
+        <View style={styles.buttonWrapper1}>
+          <Button
+            title="Go to Explore"
+            onPress={() => navigateTo(navigation, 'Explore')}
```

Comment:
"Explore", "Review", "Quiz" ... should be put into a file called ConstantName.js, then from exports const, we have stable names in our project.

Reply:
great suggestion